flag_engine/segments/types.py

```diff
@@ -0,0 +1,24 @@
+from typing import Literal
```

Comment:
Oh nice, I didn't know you could do this!

Reply:
Pydantic is awesome :)